Fixing various spelling errors (#28569)
numpy/numpy: doc/release/upcoming_changes/28569.change.rst

> * A spelling error in the error message returned when converting a string to a float with the
> method ``np.format_float_positional`` has been fixed.

Reviewer: ```suggestion
* A spelling error in the error message returned when converting a string to a float with the
  method ``np.format_float_positional`` has been fixed.
```

> Rewording eError messages when converting strings to floats
> -----------------------------------------------------------
> The error messages returned when converting a string to a float with the
> method ``np.format_float_positional`` have been slightly reworded. There
> was a spelling fix from "formating" to "formatting".

Reviewer: ```suggestion
* A spelling error in the error message returned when converting a string to a float with the
method ``np.format_float_positional`` has been fixed.
```
